bot/discord/commands/server/list.js

```diff
@@ -43,7 +43,12 @@ exports.run = async(client, message, args) => {
                     loadingMsg.delete();
                 }, 500)
             }, 5000)
-        }).catch(err => {});
+        }).catch(err => {
```

Comment:
Why won't you remove unnecessary `err` argument?
```suggestion
        }).catch(() => {
```